dev-packages/browser-integration-tests/suites/public-api/logger/subject.js

```diff
@@ -0,0 +1,6 @@
+Sentry.logger.info('test info');
+Sentry.logger.debug('test debug');
+Sentry.logger.warn('test warn');
+Sentry.logger.error('test error');
+Sentry.logger.fatal('test fatal');
+Sentry.logger.critical('test critical');
```

Comment:
Should we maybe also test the overloaded functions here?

Reply:
Overloaded functions only exist in Node

But we should test parameterization, I can add those!